src/App.js

```diff
@@ -1,19 +1,37 @@
-import { BrowserRouter as Router, Routes, Route, BrowserRouter } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route, BrowserRouter,useMatch} from "react-router-dom";
 import Projects from "./pages/Projects/index";
 import NavBar from "./components/NavBar";
 import Home from "./pages/Home/index";
 import Absence from "./pages/Absence/index"
 import Employee from "./pages/Employes/index";
 import Sidebar from "./components/Sidebar";
+import Login from "./pages/Login/Login";
 import Style from "./style.module.scss"
 import "./reset.scss"
+import "./App.scss";
 
 function App() {
-  // console.log(window.location.pathname)
+  const match= useMatch("/")
 
-  // if (window.location.pathname === '/login') {
-  //   return 1
-  // }
+  console.log(window.location.pathname)
+
+  if (window.location.pathname === '/') {
+    return (
+      <div style={{textAlign:'center'}}>
+       <NavBar/>
+          
+           {!match&&<Sidebar/>}
+           <Routes>
+           <Route path="/" element={<Login />} />
+             <Route path="/Home" element={<Home/>}/>
+             <Route path="/Projects" element={<Projects />} />
+             <Route path="/Absence" element={<Absence />} />
+             <Route path="/Employes" element={<Employee />} />
+           </Routes>
+        
+      </div>
```

Comment:
```suggestion
      <div style={{textAlign:'center'}}>
       <NavBar/>
           <Login />
      </div>
```

This should be enough, we do not need the other pages or the Routes we the user is on the login page